core/src/main/java/org/mule/runtime/core/internal/policy/DefaultPolicyManager.java

```diff
@@ -80,13 +90,18 @@ public SourcePolicy createSourcePolicyInstance(Component source, CoreEvent sourc
                                                                                                                                             .getSuccessfulExecutionResponseParametersFunction()
                                                                                                                                             .apply(flowExecutionResult),
                                                                                                                                         messageSourceResponseParametersProcessor)))
+          .doOnNext(result -> logSourcePolicySuccessfullResult(result.getRight()))
```

Comment:
Same thing here... you are already logging at the end of the CompositeSourcePolicy, these logging here seems redundant